### PR TITLE
Add GitHub action

### PR DESCRIPTION
**Pull request by jazeee**

### CODE REVIEW

File: ui/bin/roll20.js [hunk 1]
```diff
@@ -1,10 +1,3 @@
-(function() {
-  /**
-   * This is a test function that has no effect.
-   * Just testing the mergeScripts.sh
-   * This is also an example of an IIFE, which allows each script to be self-contained to reduce risks of collisions between scripts.
-   */
-})();
 // eslint-disable-next-line no-unused-vars
 const prevEPRoll = function() {
```
Comment (jazeee) [inline ui/bin/roll20.js:2]: It worked! Unsure why it rearranged things, but I am guessing we can adjust the `find` script to be case insensitive order.

**Comment by johnlamping** [comment]
Nice! I hadn't seen the IIFE pattern before.

**Comment by johnlamping** [comment]
Do you want to merge this automatically?

**Comment by jazeee** [comment]
Merged into the upstream branch.

**Pull request merged**